pkg/volume/csi/csi_plugin_test.go

```diff
@@ -655,23 +685,31 @@ func TestPluginNewMounterWithInline(t *testing.T) {
 			spec:          volume.NewSpecFromPersistentVolume(makeTestPV("test-pv1", 20, testDriver, testVol), true),
 			podUID:        types.UID(fmt.Sprintf("%08X", rand.Uint64())),
 			namespace:     "test-ns1",
-			csiVolumeMode: persistentVolumeMode,
+			csiVolumeMode: storagev1beta1.PersistentVolumeMode,
 		},
 		{
 			name:          "mounter with volume source",
 			spec:          volume.NewSpecFromVolume(makeTestVol("test-vol1", testDriver)),
 			podUID:        types.UID(fmt.Sprintf("%08X", rand.Uint64())),
 			namespace:     "test-ns2",
-			csiVolumeMode: ephemeralVolumeMode,
+			csiVolumeMode: storagev1beta1.EphemeralVolumeMode,
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			plug, tmpDir := newTestPlugin(t, nil)
+			// The fake driver currently supports all modes.
+			volumeModes := []storagev1beta1.CSIVolumeMode{
```

Comment:
should we make this configurable per test case so we can cover all of the scenarios in `supportsVolumeMode`?

Reply:
All test cases are now tested against a driver that supports both modes, only ephemeral and only persistent.